activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -774,6 +774,11 @@ def quoted_scope(name = nil, type: nil)
             scope
           end
 
+          def relkind
```

Comment:
:thinking: There's already space before interpolation at `WHERE i.relkind #{relkind}`. There's no need to return string starting with space.

Reply:
If we hardcode the IN clause, we can get rid of this method 😉